build_projects/dotnet-host-build/build.sh

```diff
@@ -85,7 +85,9 @@ done < "$REPOROOT/branchinfo.txt"
 [ -z "$DOTNET_INSTALL_DIR" ] && export DOTNET_INSTALL_DIR=$REPOROOT/.dotnet_stage0/$(uname)
 [ -d "$DOTNET_INSTALL_DIR" ] || mkdir -p $DOTNET_INSTALL_DIR
 
-$REPOROOT/scripts/obtain/dotnet-install.sh --channel $CHANNEL --verbose
+
+DOTNET_INSTALL_SCRIPT_URL="https://raw.githubusercontent.com/dotnet/cli/b939c88e0cf7c2e204f91a5161acdd2f5a732bf9/scripts/obtain/dotnet-install.sh"
```

Comment:
It's unlikely that we would make such a drastic install script change, and if we do I hope we maintain some level of backcompat so I'll go ahead and take latest. 

While I agree that there should be only one version of the CLI in use, I don't think it makes sense to figure out how to integrate these currently since pretty soon we'll only be using buildtools.